### PR TITLE
Add a release build with RTTI in CircleCI

### PR DESCRIPTION
**Pull request by siying**
Summary:
Release build RTTI is not covered in CI. Add one.

Test Plan: Watch the build results.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,6 +47,20 @@ jobs:
       - run: make V=1 -j32 release | .circleci/cat_ignore_eagain
       - run: ./db_stress --version # ensure with gflags
 
+  build-linux-release-rtti:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: 2xlarge
```
Comment (jay-zhuang) [inline .circleci/config.yml:53]: should we use `large` host to save cost?

**Review by jay-zhuang — APPROVED** [review]
LGTM

**Review by riversand963 — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (siying) [reply .circleci/config.yml]: Good idea. Let me change to that.

**Comment by siying** [comment]
The CI failures are not related for sure.

**Pull request closed without merging**